gcp/modules/gcp-stackdriver-monitoring/alert_k8s_snapshots_couchdb.tf

```diff
@@ -2,39 +2,47 @@ resource "google_monitoring_alert_policy" "k8s_snapshots_couchdb" {
   display_name = "Snapshots are being created for persistent volumes of CouchDB stateful set"
   combiner     = "OR"
 
-  conditions = [{
-    condition_absent {
-      filter   = "metric.type=\"logging.googleapis.com/user/k8s_snapshots.couchdb.snapshot_created\" resource.type=\"k8s_container\""
-      duration = "300s"
-
-      aggregations {
-        alignment_period   = "600s"
-        per_series_aligner = "ALIGN_SUM"
-        group_by_fields    = []
-      }
-    }
-
-    display_name = "Snapshot creation events are missing in K8s-snapshot logs for CouchDB"
-  },
+  conditions = [
     {
       condition_absent {
         filter   = "metric.type=\"logging.googleapis.com/user/compute.disks.createSnapshot\" resource.type=\"gce_disk\""
         duration = "300s"
 
         aggregations {
-          alignment_period   = "600s"
+          alignment_period   = "300s"
           per_series_aligner = "ALIGN_SUM"
           group_by_fields    = []
```

Comment:
I believe `cross_series_reducer` might need to be set, otherwise, the  `group_by_fields` will be ignored.

Reply:
Yeah, documentation is pretty vague about `cross_series_reducer` usage, and I don't observe any difference in condition behavior with and without it. However, if I try to configure same condition manually using UI, it completes `crossSeriesReducer` field (aggregation selector in UI) [automatically](https://app.google.stackdriver.com/metrics-explorer?project=gpii-gcp-prd&timeSelection=%7B%22timeRange%22:%221d%22%7D&xyChart=%7B%22dataSets%22:%5B%7B%22timeSeriesFilter%22:%7B%22filter%22:%22metric.type%3D%5C%22logging.googleapis.com%2Fuser%2Fcompute.disks.createSnapshot%5C%22%20resource.type%3D%5C%22gce_disk%5C%22%22,%22perSeriesAligner%22:%22ALIGN_SUM%22,%22crossSeriesReducer%22:%22REDUCE_SUM%22,%22secondaryCrossSeriesReducer%22:%22REDUCE_NONE%22,%22minAlignmentPeriod%22:%2260s%22,%22groupByFields%22:%5B%22resource.label.%5C%22zone%5C%22%22%5D,%22unitOverride%22:%221%22%7D,%22targetAxis%22:%22Y1%22,%22plotType%22:%22LINE%22%7D%5D,%22options%22:%7B%22mode%22:%22COLOR%22%7D,%22constantLines%22:%5B%5D,%22timeshiftDuration%22:%220s%22,%22y1Axis%22:%7B%22label%22:%22y1Axis%22,%22scale%22:%22LINEAR%22%7D%7D&isAutoRefresh=true). So I think it is safer to add it, thanks for noticing!